Modules/Core/Common/test/itkBoundingBoxTest.cxx

```diff
@@ -205,6 +206,21 @@ int itkBoundingBoxTest (int, char*[])
     }
   }
 
+  // Expect four corners for a two-dimensional box:
```

Comment:
Before this line, I think you should do `#ifndef ITK_LEGACY_REMOVE`

Reply:
done